samples/app/src/main/java/com/sourcepoint/app/v6/MainActivityKotlin.kt

```diff
@@ -229,6 +229,7 @@ class MainActivityKotlin : AppCompatActivity() {
                     when (message.campaignType) {
                         CampaignType.GDPR -> dataProvider.gdprPmId
                         CampaignType.CCPA -> dataProvider.ccpaPmId
+                        CampaignType.USNAT -> throw RuntimeException()
```

Comment:
I think we would strive more from:

> `// TODO ...`

since we can check the list of `TODO's` in order to not forget to update such places